src/hotspot/os/windows/os_windows.cpp

```diff
@@ -6281,3 +6281,106 @@ const void* os::get_saved_assert_context(const void** sigInfo) {
   *sigInfo = nullptr;
   return nullptr;
 }
+
+/*
+ * Windows/x64 does not use stack frames the way expected by Java:
```

Comment:
Should that comment say /x64 if it's for all aarchs?

Reply:
> Should that comment say /x64 if it's for all aarchs?

The behavior described in this comment section applies to x64 only (it is the original motivation for introducing this platform printing implementation). This change recognizes that the preexisting Windows x64 solution is essentially platform agnostic (modulo the machine type and registers) and therefore reuses it for Windows AArch64.